shell/initialize/App.vue

```diff
@@ -78,43 +60,6 @@ export default {
         this.isOnline = window.navigator.onLine;
       }
     },
-
-    async refresh() {
```

Comment:
This isn't being used anywhere. I checked the entire app since we give access to it via `$globalApp`.

Reply:
I gave this a quick review and saw the same things. I also think this looks safe to remove.